api/handler/delete.go

```diff
@@ -213,7 +213,10 @@ func (h *handler) DeleteMultipleObjectsHandler(w http.ResponseWriter, r *http.Re
 
 	var errs []error
 	for _, obj := range deletedObjects {
-		if obj.Error == nil {
+		if obj.Error == nil || (len(obj.VersionID) == 0 && errors.IsS3Error(obj.Error, errors.ErrInvalidVersion)) {
```

Comment:
When we try to delete not existing objects without specified version:
- aws responses without any error in any mode: quiet or verbose 
- the gate responses with an error `invalid version is specified`
- tests requires no error in response

That's why I decided to consider `errors.ErrInvalidVersion` as noError if `versionID` is not specified in a request.

Reply:
Maybe we can return no error instead of `errors.ErrInvalidVersion` from `layer` in this case?

Reply:
Yep, I thought about it, but I think we need to check how changing of logics will influence in other functions which use `deleteObjects` function. Maybe later after fixing other tests.